quarkchain/cluster/slave.py

```diff
@@ -1390,6 +1390,8 @@ def parse_args():
 
 
 def main():
+    from quarkchain.cluster.jsonrpc import JSONRPCWSServer
```

Comment:
import line should on the top of the file

Reply:
This line is in `main()` because `SlaveServer` in this file is imported in `jsonrpc.py` as well. These two files may not be able to import each other.

Reply:
ah I see